parm/ocnicepost/ice_sfs.csv

```diff
@@ -0,0 +1,63 @@
+! variable name,dimension,grid location,remapping method,vector pair,grid location of vector pair,gb2 varname,discription,unit,gc1,gc2,gc3,gc4,gc5,gc6,gc7,gc8
+   		'Tsfc_h',2,'Ct' , 'bilinear' , '', '','','snow/ice surface temperature' ,'C' ,0,0,0,0,0,0,0,0
+   		'aice_h',2,'Ct' , 'bilinear' , '', '','','ice area  (aggregate)' ,'1' ,0,0,0,0,0,0,0,0
+   		'uvel_h',2,'Bu_x' , 'bilinear' ,'vvel_h','Bu_y','','ice velocity (x)','m/s' ,0,0,0,0,0,0,0,0
+   		'vvel_h',2,'Bu_y' , 'bilinear' ,'uvel_h','Bu_x','','ice velocity (y)','m/s' ,0,0,0,0,0,0,0,0
+                'sice_h',2,'Ct', 'bilinear' ,'','','', 'bulk ice salinity' , 'ppt' ,0,0,0,0,0,0,0,0
+                'fswdn_h',2,'Ct','conserve','','','', 'down solar flux' , 'W/m^2' ,0,0,0,0,0,0,0,0
+                'flwdn_h',2,'Ct','conserve','','','', 'down longwave flux' , 'W/m^2' ,0,0,0,0,0,0,0,0
+                'snow_h',2,'Ct', 'bilinear' ,'','','', 'snowfall rate (cpl)' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'rain_h',2,'Ct', 'bilinear' ,'','','', 'rainfall rate (cpl)' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'sst_h',2,'Ct', 'bilinear' ,'','','', 'sea surface temperature' , 'C' ,0,0,0,0,0,0,0,0
+                'sss_h',2,'Ct', 'bilinear' ,'','','', 'sea surface salinity' , 'ppt' ,0,0,0,0,0,0,0,0
+                'uocn_h',2,'Bu_x', 'bilinear' ,'vocn_h','Bu_y','', 'ocean current (x)' , 'm/s' ,0,0,0,0,0,0,0,0

```

Comment:
The ocean currents in the CICE history files are not on the Bu (corner points). Please check a native ice history file. They are located on the center (Ct) grid location.

```
float vocn_1(time, nj, ni) ;
		vocn_1:units = "m/s" ;
		vocn_1:long_name = "ocean current (y)" ;
		vocn_1:coordinates = "TLON TLAT time" ;
```